lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,72 @@
+# Gerrit
+
+## Authentication
+
+<figure markdown>
+  ![Gerrit HTTP access token](/assets/images/gerrit-http-password.png){ loading=lazy }
+  <figcaption>First, create a HTTP access token for the Renovate account.</figcaption>
+</figure>
+
+Let Renovate use your HTTP access token by doing _one_ of the following:
```

Comment:
do we really need this basics here?

Reply:
I think yes, this http-token will not be used normally from Gerrit users.